app/static/darwin/github.sh

```diff
@@ -5,8 +5,8 @@ function realpath() {
   /usr/bin/perl -e "use Cwd;print Cwd::abs_path(@ARGV[0])" "$0";
 }
 
-CONTENTS="$(dirname "$(dirname "$(dirname "$(dirname "$(realpath "$0")")")")")"
-BINARY_NAME="$(ls "$CONTENTS/MacOS/")"
+CONTENTS="$(command dirname "$(command dirname "$(command dirname "$(command dirname "$(realpath "$0")")")")")"
+BINARY_NAME="$(TERM=dumb command ls "$CONTENTS/MacOS/")"
```

Comment:
I wanted to use `--color=none` as suggested in https://github.com/desktop/desktop/issues/13935#issuecomment-1050087225, but `--color` is not available on macOS older than Monterey. I _think_ just setting `TERM=dumb` achieves the same result, I'm waiting to see if @offirgolan can confirm 🤞

Reply:
That worked for me :)